docs/en/advanced/Alluxio-Storage-Management.md

```diff
@@ -100,6 +100,16 @@ alluxio.worker.tieredstore.level0.dirs.quota=16GB,100GB,100GB
 
 Note that the ordering of the quotas must match with the ordering of the paths.
 
+We also need to specify what kind of storage medium each path is on. Our previous example had one path that is ramdisk, followed by two SSDs. We can indicate that by using:
+
+```
+alluxio.worker.tieredstore.level0.dirs.mediumtype=MEM,SSD,SSD
```

Comment:
Should this be updated in the other code blocks?

Reply:
moved to the other code block. 
PTAL